### PR TITLE
fix: update and shift unittest

### PR DESCRIPTION
**Pull request by darshankabariya**

### CODE REVIEW

**Comment by darshankabariya** [comment]
Hi @rymnc,
In my previous PR, I missed your last comment. I've now addressed it and find better place for the test.

File: tests/node/test_wakunode_relay_rln.nim [hunk 1]
```diff
@@ -136,6 +137,33 @@ suite "Waku RlnRelay - End to End":
...
+        onFatalErrorAction: proc(errStr: string) =
+          raiseAssert errStr
+        ,
+      )
+
+      try:
+        await node.mountRlnRelay(wakuRlnConfig)
+      except CatchableError as e:
+        check e.msg ==
+          "failed to mount WakuRlnRelay: rln-relay-user-message-limit can't be exceed then MAX_MESSAGE_LIMIT set by rln contract"
```
Comment (rymnc) [inline tests/node/test_wakunode_relay_rln.nim:165]:
```suggestion
          "failed to mount WakuRlnRelay: rln-relay-user-message-limit can't exceed the MAX_MESSAGE_LIMIT in the rln contract"
```

**Review by rymnc — APPROVED** [review]
LGTM, minor nit, will need to change the error returned :)

**Review by Ivansete-status — APPROVED** [review]
Thanks for that! LGTM as long as all comments are addressed :partying_face:

**Pull request merged**